# This mesage defines what objects are attached to a link. The objects
# are assumed to be defined in the reference frame of the link they
# are attached to

string robot_name
string link_name
collision_map/CollisionMap objects
